src/server/game/Globals/ObjectMgr.cpp

```diff
@@ -1101,6 +1110,13 @@ void ObjectMgr::LoadCreatureAddons()
             creatureAddon.emote = 0;
         }
 
+        if (creatureAddon.visibilityDistanceType >= VisibilityDistanceType::Max)
+        {
+            TC_LOG_ERROR("sql.sql", "Creature (GUID: " UI64FMTD ") has invalid visibilityDistanceType (%u) defined in `creature_addon`.",
+                guid, AsUnderlyingType(creatureAddon.visibilityDistanceType));
```

Comment:
I hope you have got a plan for dealing with this feedback from travis-ci :
```cpp
/home/travis/build/TrinityCore/TrinityCore/src/server/game/Globals/ObjectMgr.cpp:1116:17:
 fatal error: 
      format specifies type 'unsigned long' but the argument has type
      'ObjectGuid::LowType' (aka 'unsigned int') [-Wformat]
                guid, AsUnderlyingType(creatureAddon.visibilityDistanceType));
~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
fixed

Reply:
(you may mark this conversation as resolved if you want)